src/chats/components/conversation/footer/MessageComposer.tsx

```diff
@@ -560,6 +563,8 @@ const MessageComposer: React.FC<ConversationMessageComposerProps> = ({ roomId })
 				/>
 			)}
 		</Container>
+	) : (
+		<ForwardFooter roomId={roomId} />
 	);
```

Comment:
I think it's better to choose what type of footer display (if `MessageCompose` or `ForwardFooter`) in `ConversationFooter.tsx`